source/scpd/types/XDRBase.d

```diff
@@ -88,34 +91,14 @@ class xdr_wrong_union : logic_error
 
 extern(D) static immutable XDR_MAX_LEN = 0xffff_fffc;
 
-//struct xarray(T, uint32_t N) { public array!(T, N) data; alias data this; }
-struct xarray(T, uint32_t N)
+///
+struct opaque_array(uint32_t N = XDR_MAX_LEN)
```

Comment:
It's only used for `Hash` right ?

Reply:
Also `uint256`, `SignatureHint` and some other types I guess. What are you thinking?